xml/System.Net.PeerToPeer.Collaboration/PeerApplication.xml

```diff
@@ -452,7 +452,7 @@
       <Docs>
         <param name="obj">The <see cref="T:System.Object" /> to test for equality.</param>
         <summary>Compares the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> to the specified <see cref="T:System.Object" />.</summary>
-        <returns>Returns <see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> is equal to the specified object instance, else <see langword="false" />.</returns>
+        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> is equal to the specified object instance, else <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> is equal to the specified object instance; otherwise, <see langword="false" />.</returns>
```